### ISSUE TITLE
loglevel in configuration file being ignored

### ISSUE THREAD

**Issue by tunecrew** [pgjones/hypercorn#63]
I'm configuring hypercorn w/ a toml file that is loaded when I invoke hypercorn from docker:
```
ENTRYPOINT ["/usr/local/bin/hypercorn", "--config", "/project/conf/hypercorn.toml", "/project/src/engine.asgi:application"]
```
The logging section of hypercorn.toml is this:
```
# logging
accesslog = '-'
errorlog = '-'
debug = false
loglevel = 'WARNING'
```
I'm still getting very noisy INFO level logging at stdout though at every request, e.g.:
```
[2022-08-25 23:40:58 +0000] [1] [INFO] [172.31.0.1:63838](http://172.31.0.1:63838/) - - [25/Aug/2022:23:40:58 +0000] "GET /api/v1/manager/users 2" 200 300 "http://localhost:3000/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.5112.102 Safari/537.36"
```
Hypercorn is acting as the ASGI server for Djagno.

Any idea how to fix this?

**Comment by pgjones** [comment]
Should be fixed in 3f804ce45be2125334a30ad169e3bde64e5c71cb

**Comment by tunecrew** [comment]
confirming this is fixed - thanks!
